### PR TITLE
adapts title capitalization to improve consistency

### PR DESCRIPTION
**Pull request by martinweck**
This is a suggestion to adapt the title capitalization to improve consistency using the following rule:
- chapter: title case
- section: title case (by the second commit as reaction to @gernotstarke's comment)
- everything below, e.g. bullets: lower case (terms like cloud native in LG 1-10 in italics)

hint:
- title case: 1st word, nouns, pronouns, verbs, adjectives and subordinate conjunctions start capitalized
- sentence case: 1st letter capitalized
- It does not matter how we do it, but the golden rule is to do whatever you do consistently. So I looked how the text has been written and derived the rule above from the current text (how chapters, sections, bullets are written right now in most of the cases)

### CODE REVIEW

**Review by gernotstarke — COMMENTED** [review]
Im happy with the changes in bullet lists.

But I'm unhappy with changing of most headings, as we deliberately migrated to upper-case couns there... and I don't like to revert those...

Therefore, it's a mixture of approval and rejection from my point-of-view.

**Comment by gernotstarke** [comment]
alternative suggestion:

* chapter: title case
* all section headings: title case
* everything below, e.g. bullets: lower case (terms like cloud native in LG 1-10 in italics)

File: docs/01-basics/01-basics-duration-terms.adoc [hunk 1]
```diff
@@ -35,7 +35,7 @@ Typen von IT-Systemen (eingebettete Systeme; Echtzeitsysteme; Informationssystem
 | Duration: 120 min. | Exercises: none
 |===
 
-=== Relevant Terms
+=== Relevant terms
```
Comment (gernotstarke) [inline docs/01-basics/01-basics-duration-terms.adoc:38]: please leave as "Relevant Terms"

File: docs/02-design/01-design-duration-terms.adoc [hunk 2]
```diff
@@ -37,7 +37,7 @@ Domain-Driven Design
 | Duration: 330 min. | Excercises: 90 min.
 |===
 
-=== Relevant Terms
+=== Relevant terms
```
Comment (gernotstarke) [inline docs/02-design/01-design-duration-terms.adoc:40]: again, Relevant Terms

File: docs/04-quality/01-quality-duration-terms.adoc [hunk 3]
```diff
@@ -19,7 +19,7 @@ Qualität; Qualitätsmerkmale; DIN/ISO 25010; Qualitätsszenarien; Qualitätsbau
 | Duration: 60 min. | Exercises: 60 min.
 |===
 
-=== Relevant Terms
+=== Relevant terms
```
Comment (gernotstarke) [inline docs/04-quality/01-quality-duration-terms.adoc:22]: Relevant Terms

**Comment by martinweck** [comment]
Thank you for your feedback @gernotstarke . Relevant terms is one level below the chapter level, i.e. section level. So I would suggest if we change the section level to title case, then we should do this for all sections to avoid inconsistencies. Therefore, sections like the following would be affected, too:
E.g. in chapter introduction
- What does a Foundation Level Training Convey?
- Out of Scope
- Structure, Duration and Teaching Methods
- Learning Goals and Relevance for the Examination etc.

Also "Learning Goals" should be in Title Case if that is the case for Relevant Terms

I will adapt the MR this evening accordingly, then you can check if this fits better your expectations.

**Comment by martinweck** [comment]
@gernotstarke Thank you for your feedback. I have adapted all section heading to title case, hence, `Relevant Terms`, too. I hope that is better.

**Review by gernotstarke — APPROVED** [review]
thanx for making things a little better ...

**Pull request merged**